src/System Application/App/Email/src/Email/EmailDispatcher.Codeunit.al

```diff
@@ -89,13 +87,13 @@ codeunit 8888 "Email Dispatcher"
                 UpdateOutboxError(LastErrorText, EmailOutbox);
                 UpdateOutboxStatus(EmailOutbox, EmailOutbox.Status::Failed);
 
-                // if email is not rescheduled, it means it has exceeded the retry limit, stop retrying
-                if ClientTypeManagement.GetCurrentClientType() = CLIENTTYPE::Background then begin
+                if EmailOutbox."Is Background Task" then begin
                     if EmailOutbox."Retry No." = 1 then
                         EmailRetryImpl.CreateEmailRetry(EmailOutbox);
 
                     EmailRetryImpl.UpdateEmailRetryRecord(EmailOutbox."Message Id", EmailOutbox."Retry No.", EmailOutbox.Status::Failed, LastErrorText, EmailOutbox."Date Queued", EmailOutbox."Date Failed", EmailOutbox."Date Sending");
 
+                    // if email is not rescheduled, it means it has exceeded the retry limit, stop retrying
                     if RetrySendEmail(EmailOutbox) then exit;
```

Comment:
Is this ever `true`? Redundant to wrap it in if/else also when it's not a `TryFunction`.